docs/platform/howto/manage-org-users.rst

```diff
@@ -0,0 +1,59 @@
+Manage users in an organization
+================================
+
+.. important::
+    Organization users are available as a :doc:`feature preview </docs/platform/howto/feature-preview>`.
+
+Adding users to your organization lets you give them access to specific organizational units, projects, and services within that organization. 
+
+Invite users to an organization
+---------------------------------
+
+To add users to your organization, send them an invite:
+
+#. Click **Admin**.
+
+#. Click **Users**.
+
+#. Click **Invite users**.
+
+#. Enter the email addresses of the people you want to invite. 
+
+#. To automatically add these users to user groups, switch the toggle to **Yes** and select the groups.
+
+#. Click **Invite users**.
+
+The users receive an email with instructions to sign up (for new users) and accept the invite.
+
+
+Remove users from an organization
+----------------------------------
+
+To remove a user from an organization: 
+
+#. Click **Admin**.
+
+#. Click **Users**.
+
+#. Find the user that you want to remove and click the **Actions** menu. 
+
+#. Select **Remove**.
+
+#. Confirm you want to remove the user by clicking **Remove user**.
```

Comment:
Let's add a sentence that the user will be removed from all teams and projects and will have no longer access to any resource in the Org.